Packs/CloudflareWAF/pack_metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "Cloudflare WAF",
+    "description": "Use Cloudflare WAF to manage firewall rules, filters, and IP-lists.",
+    "support": "xsoar",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "categories": [
+        "Network Security"
+    ],
+    "tags": [],
+    "useCases": [],
+    "keywords": []
+}
```

Comment:
please add the 'marketplaces' field and populate it with the marketplaces this pack is going to be supported by (see example [here](https://github.com/demisto/content/blob/master/Packs/AMP/pack_metadata.json))